### PR TITLE
Temp change

### PR DESCRIPTION
**Pull request by aguibert**
Sample text in PR body

UPDATED

Run LIBERTY_PAYARA first

### CODE REVIEW

**Comment by aguibert** [comment]
Sample PR comments

File: .github/GenerateCategories.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    static final boolean DEBUG = false;
+
+    public static void main(String args[]) throws Exception {
+
+        debug("Starting");
+
+        debug("Inputs are: " + Arrays.toString(args));
+        String prSummary = null;
+        if (args != null && args.length > 0)
+            prSummary = args[0].toUpperCase();
```
Comment (KyleAure) [inline .github/GenerateCategories.java:34]: It looks like we don't actually pass the PR Summary arg to this method.
Or did I miss it somewhere?

File: .github/workflows/microshed-ci.yml [hunk 2]
```diff
@@ -14,68 +17,48 @@ on:
...
-          ${{ runner.os }}-gradle-
-    - name: Build with Gradle
+    - name: Generate output
+      id: gen-params
       run: |
-        chmod +x gradlew
-        ./gradlew assemble testClasses
+        echo "::set-output name=test-os::ubuntu-latest"
+        echo "::set-output name=test-java::11"
+        MATRIX_RESULT=$(java .github/GenerateCategories.java)
```
Comment (KyleAure) [inline .github/workflows/microshed-ci.yml:44]: ^^ This is where I would expect to see the prSummary info passed

File: .github/GenerateCategories.java [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+        // Discover all categories by checking for files under .github/test-categories/
+        SortedSet<String> allCategories = new TreeSet<>();
+        for (File categoryFile : Paths.get(".github/test-categories").toFile().listFiles()) {
+            allCategories.add(categoryFile.getName().toUpperCase());
+        }
+        debug("All discovered categories are: " + allCategories);
+
+        // Any categories that were mentioned in the PR summary should run first
+        TreeMap<Integer, String> menteiondCategories = new TreeMap<>();
```
Comment (KyleAure) [inline .github/GenerateCategories.java:45]: Spelling: mentionedCategories

**Pull request closed without merging**